app/src/test/java/org/greenstand/android/TreeTracker/utils/FakeFileGenerator.kt

```diff
@@ -191,4 +194,37 @@ object FakeFileGenerator {
         value = "random value",
         treeCaptureId = 1212
     )
+
+    val fakeProgressGreen = DepthButtonColors(
+        color = AppColors.Green,
+        shadowColor = AppColors.GreenShadow,
+        disabledColor = AppColors.GreenDisabled,
+        disabledShadowColor = AppColors.GreenShadowDisabled
+    )
+
+    val fakeMessagePurple = DepthButtonColors(
+        color = AppColors.Purple,
+        shadowColor = AppColors.PurpleShadow,
+        disabledColor = AppColors.PurpleDisabled,
+        disabledShadowColor = AppColors.PurpleShadowDisabled
+    )
+
+    val fakeUploadOrange = DepthButtonColors(
+        color = AppColors.Orange,
+        shadowColor = AppColors.OrangeShadow,
+        disabledColor = AppColors.GrayShadow,
+        disabledShadowColor = AppColors.GrayShadow
+    )
+    val fakeYellow = DepthButtonColors(
+        color = AppColors.Yellow,
+        shadowColor = AppColors.YellowShadow,
+        disabledColor = AppColors.GrayShadow,
+        disabledShadowColor = AppColors.GrayShadow
+    )
+    val fakeSkyBlue = DepthButtonColors(
+        color = AppColors.SkyBlue,
+        shadowColor = AppColors.SkyBlueShadow,
+        disabledColor = AppColors.GrayShadow,
+        disabledShadowColor = AppColors.GrayShadow
+    )

```

Comment:
There is no need to make test colors. Just use the AppButtonColors class.

Reply:
done